.env.sample

```diff
@@ -8,5 +8,10 @@ GNOSIS_DB_URL=
 ETHEREUM_NODE_URL=
 GNOSIS_NODE_URL=
 
+# credentials for writing to db connection
+SOLVER_SLIPPAGE_HOST=
```

Comment:
Why do we need three variables here while we only need one variable for the backend dbs? (E.g., see lines 4 and 5)